src/main/java/com/qaprosoft/carina/demo/gui/hasiuk/pages/PhonePage.java

```diff
@@ -0,0 +1,66 @@
+package com.qaprosoft.carina.demo.gui.hasiuk.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.demo.gui.hasiuk.components.phone.page.Comment;
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PhonePage extends GsmArenaPage {
+
+    @FindBy(xpath = "//h1")
+    private ExtendedWebElement articleName;
+
+    @FindBy(xpath = "//li[@class='article-info-meta-link light'][4]")
+    private ExtendedWebElement opinionsTab;
```

Comment:
you should use unique xpath like this (//li[@class='article-info-meta-link light'])[4]